### PR TITLE
register akshya.is-a.dev domain

### PR DESCRIPTION
**Pull request by LowkeyGUY4444**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://lowkeyguy4444.github.io/

### CODE REVIEW

**Comment by LowkeyGUY4444** [comment]
what is wrong with my request.

File: domains/_github-pages-challenge-LowkeyGUY4444.akshya.json
Comment (Stef-00012) [inline domains/_github-pages-challenge-LowkeyGUY4444.akshya.json:1]: `akshya` doesn't exist so you can't add a subdomain of it

File: domains/akshya.is-a.dev.json
Comment (Stef-00012) [inline domains/akshya.is-a.dev.json:1]: file name can not contain `.is-a.dev`

**Review by Stef-DP — DISMISSED** [review]

**Comment by LowkeyGUY4444** [comment]
now file renamed
can you check again

**Comment by LowkeyGUY4444** [comment]
can you check again

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**